presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -79,6 +79,8 @@
 
 public class TestSelectiveOrcReader
 {
+    private static final int NUM_ROWS = 31_234;
```

Comment:
@mbasmanova Why not 30,000?

Reply:
30_000 makes exactly 3 row groups; the new number allows for testing "partial" row groups.